packages/fuel-connector/src/prices/FuelPricesContractAdapter.ts

```diff
@@ -9,14 +9,19 @@ export class FuelPricesContractAdapter implements IPricesContractAdapter {
     private gasLimit: number
   ) {}
 
+  protected static extractNumbers(result: InvocationResult): number[] {
+    // eslint-disable-next-line
+    return result.value.map(num);
+  }
```

Comment:
You can tell `InvocationResult` what you expect from it as a value property.
```suggestion
  protected static extractNumbers(
    result: InvocationResult<U256Output[]>
  ): number[] {
    return result.value.map(num);
  }
```